### PR TITLE
new file:   packages/orrandomForest/orrandomForest.3.0.0/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**
New executable: orrf_model

### CODE REVIEW

File: packages/orrandomForest/orrandomForest.3.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+description: """
+Uses the R randomForest package under the carpet.
+This package really fires up and talks to an R interpreter.
+Data are exchanged via text files.
+For details, cf.
+Breiman, L., 2001. Random forests. Machine learning, 45(1), pp.5-32
+(DOI = 10.1023/A:1010933404324)."""
+extra-files: [
+  ["install_randomForest.r" "md5=c39ce051de6937deb5095a1906d86c0c"]
+  ["install_matrix.r" "md5=ab1c0ae726388159b1315bc9fe61a013"]
```
Comment (kit-ty-kate) [inline packages/orrandomForest/orrandomForest.3.0.0/opam:44]: These files are missing from the release

**Comment by UnixJunkie** [comment]
ok, I will add them

**Comment by UnixJunkie** [comment]
done

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**